### ISSUE TITLE
No functionality triggered when running the binaries of npm packages, installed locally or globally

### ISSUE THREAD

**Issue by gouravkhator** [gouravkhator/alacritty-auto-config#2]
# Issue:

When I install the npm package globally or locally, I don't see any functionality triggered.

## Issue Detailed:

If I install the package globally like the below command:

```sh
npm i -g alacritty-auto-config
```

Or locally to my other node project as:

```sh
npm i alacritty-auto-config
``` 

Then, run the bin commands for executing that built javascript file :

```sh
aac -h
```
OR,
```sh
alacritty-auto-config -h
```
OR,

```sh
npx aac -h
```
OR,
```sh
npx alacritty-auto-config -h
```

**These 'aac' and 'alacritty-auto-config' commands (that are defined in package.json bin field) will work but no output or error message will be reflected.** 
**The commands run fine but their functionality is not triggered on global or local bin commands of the package.**

Note: The package API part works fine and the executables created for different OS platforms also work fine.

**Comment by gouravkhator** [comment]
I tried installing this package locally from my dev setup itself. 

Firstly, I cloned this repo 
Line Snippets: https://github.com/gouravkhator/alacritty-auto-config/blob/main/src/index.ts#L38-L42
Commit: [f8148bd](https://github.com/gouravkhator/alacritty-auto-config/commit/f8148bd415a49f9bd023fc4e80818d30f01ca4e2))

Then I printed two things: `import.meta?.url` and `fileURL.toString()` outside if statements to debug the if conditions, to see if `main` function is actually invoked or not.

Then I ran the `npm run build` for this npm package.
Then I created another npm project named `temp-node` like below:

```sh
mkdir temp-node
cd temp-node
npm init -y
npm i ../alacritty-auto-config
```

Then, I ran the below:
```sh
npx aac -h
```

It showed different values of `import.meta?.url` and `fileURL.toString()` , which proved that main function is not even invoked when we run the local or global npm binary.
